docs/_layouts/guides.html

```diff
@@ -13,24 +13,28 @@
                 {% for post in site.guides reversed %}
 
                 <div class="row guides-post-row">
-                    <div class="guides-post-column col">
-
-                        <div class="guides-post-content guides-content">
-                            <h2 class="guides-post-title"><a href="{{ post.url | relative_url }}"
-                                    class="guides-post-title-link">{{ post.summary_title | escape }}</a></h2>
-                            <p class="guides-post-date-mobile d-sm-block d-md-none">{{ post.date | date: date_format }}
-                            </p>
-                            <p class="guides-post-paragraph">
-                                {% if post.guide_description %}
-                                {{ post.guide_description }}
+                    <div class="media">
+                        <img src="{{ post.image }}" class="guide-image" alt="...">
```

Comment:
was there a reason for "..." as alt?

Reply:
I believe it's part of the bootstrap template. I'll fill that in now 👍